pkg/disruption/backend/sampler/remote.go

```diff
@@ -0,0 +1,501 @@
+	err := callMonitorDaemonset(ctx, kubeClient, "", false)
+	if err != nil {
+		return err
+	}
+
+	err = callRBACClusterAdmin(ctx, kubeClient, false)
+	if err != nil {
+		return err
+	}
+	err = callRBACHostaccess(ctx, kubeClient, false)
+	if err != nil {
+		return err
+	}
+	err = callServiceAccount(ctx, kubeClient, false)
+	if err != nil {
+		return err
+	}
+	return callProject(ctx, projectClient, false)
+}
+
+func createDaemonset(ctx context.Context, clientset *kubernetes.Clientset, create bool, obj *appsv1.DaemonSet) error {
+	client := clientset.AppsV1().DaemonSets(namespace)
+	var err error
+	if create {
+		_, err = client.Create(ctx, obj, metav1.CreateOptions{})
+		if apierrors.IsAlreadyExists(err) {
+			return nil
+		}
+		if err != nil {
+			return err
+		}
+		if err := wait.PollUntilContextTimeout(ctx, 10*time.Second, 5*time.Minute, true, func(context.Context) (bool, error) {
+			ds, err := client.Get(context.Background(), obj.Name, metav1.GetOptions{})
+			if err != nil {
+				framework.Logf("error getting daemonsets %v", err)
+				return false, nil
+			}
+			return ds.Status.NumberReady > 0, nil
+		}); err != nil {
+			return fmt.Errorf("daemonset %s didn't roll out: %v", obj.Name, err)
+		}
+
+	} else {
+		err = client.Delete(ctx, obj.Name, metav1.DeleteOptions{})
+		if err != nil {
+			return err
+		}
+		if err := wait.PollUntilContextTimeout(ctx, 10*time.Second, 5*time.Minute, true, func(context.Context) (bool, error) {
+			_, err := client.Get(context.Background(), obj.Name, metav1.GetOptions{})
+			if err != nil {
+				if apierrors.IsNotFound(err) {
+					return true, nil
+				}
+				framework.Logf("Error getting daemonset %s: %v", err)
+				return false, nil
+			}
+			return false, nil
+		}); err != nil {
+			return fmt.Errorf("daemonset %s didn't roll out: %v", obj.Name, err)
+		}
+	}
+
+	return err
+}
+
+func callMonitorDaemonset(ctx context.Context, clientset *kubernetes.Clientset, apiIntHost string, create bool) error {
+	name := "pod-monitor"
+	labels := map[string]string{
+		"app": name,
+	}
+	bTrue := true
+	obj := &appsv1.DaemonSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: appsv1.DaemonSetSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: labels,
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: labels,
+				},
+				Spec: corev1.PodSpec{
+					Volumes: []corev1.Volume{
+						{
+							Name: "artifacts",
+							VolumeSource: corev1.VolumeSource{
+								HostPath: &corev1.HostPathVolumeSource{
+									Path: disruptionDataPath,
+								},
+							},
+						},
+					},
+					Containers: []corev1.Container{
+						{
+							Name:  "internal-lb",
+							Image: "image-registry.openshift-image-registry.svc:5000/openshift/tests:latest",
+							Env: []corev1.EnvVar{
+								{
+									Name:  "KUBERNETES_SERVICE_HOST",
+									Value: apiIntHost,
```

Comment:
interesting.  not running this from the hostnetwork?

Reply:
Split this in two daemonsets, as internal-lb needs to run with hostNetwork and service-network should run without it